tests/test_github.py

```diff
@@ -255,6 +255,82 @@ def test_fetch_issues(self):
                          issue['data']['comments_data'][0]['reactions']['total_count'])
         self.assertEqual(issue['data']['comments_data'][0]['reactions_data'][0]['user_data']['login'], 'zhquan_example')
 
+    @httpretty.activate
```

Comment:
You should also add a more generic test to check `filter_classified` items.

Take as example this PR: https://github.com/chaoss/grimoirelab-perceval/pull/497/files

Reply:
I have modified `test_backend.py`. I have included an item in `ClassifiedFieldsBackend` which is a list that contains a dictionary with a field to filter. I have extended the list CLASSIFIED_FIELDS and modified the tests according to these changes. Let me know if that is enough or if I have to add something more sophisticated